openstf/model/objective.py

```diff
@@ -2,11 +2,13 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 import optuna
+from typing import Union, Tuple
 
+import openstf  # Needed for <loss = "openstf.metrics.metrics." + self.eval_metric>  and the line <metric = eval(loss)(test_y, forecast_y)>
```

Comment:
Good that you added a comment on why this is required. But I think we can make a more clear solution to this issue.